### ISSUE TITLE
Cannot parse cubic-bezier with negative y coordinate

### ISSUE THREAD

**Issue by papandreou** [hookhookun/parse-animation-shorthand#16]
I get a `NoComma` exception when trying to parse a `cubic-bezier` with a negative y coordinate: 

```js
require('@hookun/parse-animation-shorthand').parseSingle('3s cubic-bezier(0.68, -0.82, 0.42, 1.52) infinite alternate foo')
```

```
Uncaught:
$Error: NoComma
    at Object.exports.getCubicBezier (/path/to/project/node_modules/@hookun/parse-animation-shorthand/lib/getCubicBezier.js:20:23)
    at Object.exports.parseSingleAnimationShorthand [as parseSingle] (/path/to/project/node_modules/@hookun/parse-animation-shorthand/lib/parseSingleAnimationShorthand.js:59:53) {
  code: 'NoComma'
}
```

According to the spec it's only the x coordinates that have to be in the [0:1] interval: https://www.w3.org/TR/css-easing-1/#cubic-bezier-easing-functions

**Comment by hookhookun** [comment]
I was able to reproduce the problem. I'll fix it within a week.
Thanks for the report!

**Comment by hookhookun** [comment]
I'll close this issue when I publish the new version.

**Comment by hookhookun** [comment]
This issue has been fixed in [v0.1.4](https://github.com/hookhookun/parse-animation-shorthand/releases/tag/v0.1.4).

**Comment by papandreou** [comment]
Confirmed! Thanks a lot for the fast action! 🚀
